problem building gst-python: Run-time dependency gstreamer-plugins-base-1.0 found: NO (tried pkgconfig and cmake)

I'm trying to build gst-python using `gvsbuild` and ran into this error:

```
Run-time dependency gstreamer-plugins-base-1.0 found: NO (tried pkgconfig and cmake)

testsuite\meson.build:25:2: ERROR: No default provided for dependency <NotFoundDependency gstreamer-plugins-base-1.0: False>, which is not pkg-config, cmake, or config-tool based.

A full log can be found at C:\gtk-build\build\x64\release\gst-python\_gvsbuild-meson\meson-logs\meson-log.txt

```
The file `C:\gtk-build\build\x64\release\gst-python\_gvsbuild-meson\meson-logs\meson-log.txt` doesn't exist. There is an empty folder `C:\gtk-build\build\x64\release\gst-python-rel`.

I set my PATH to find gstreamer, and can run this pipeline successfully, so I know at least that the gstreamer binaries are found:

`gst-launch-1.0.exe videotestsrc ! autovideosink`

Any help is appreciated.

No. It fails with this:

```
[57/774] Generating gst-libs/gst/tag/GstTag-1.0.gir with a custom command (wrapped by meson to set PATH, to set env)
FAILED: gst-libs/gst/tag/GstTag-1.0.gir
"d:\GitClones\gvsbuild\.venv\Scripts\python.exe" "C:\gtk-build\tools\meson-1.0.0\meson.py" "--internal" "exe" "--unpickle" "C:\gtk-build\build\x64\release\gst-plugins-base\_gvsbuild-meson\meson-private\meson_exe_python.exe_0a74008b99ac007e58c61686508356e9e5408160.dat"
while executing ['d:\\GitClones\\gvsbuild\\.venv\\Scripts\\python.exe', 'C:/gtk-build/gtk/x64/release/bin/../bin/g-ir-scanner', '--quiet', '--no-libtool', '--namespace=GstTag', '--nsversion=1.0', '--warn-all', '--output', 'gst-libs/gst/tag/GstTag-1.0.gir', '--add-init-section=extern void gst_init(gint*,gchar**);g_setenv("GST_REGISTRY_DISABLE", "yes", TRUE);g_setenv("GST_REGISTRY_1.0", "C:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gir_empty_registry.reg", TRUE);g_setenv("GST_PLUGIN_PATH_1_0", "", TRUE);g_setenv("GST_PLUGIN_SYSTEM_PATH_1_0", "", TRUE);gst_init(NULL,NULL);', '--quiet', '--c-include=gst/tag/tag.h', '-IC:/gtk-build/build/x64/release/gst-plugins-base/gst-libs/gst/tag', '-IC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gst-libs/gst/tag', '-IC:/gtk-build/build/x64/release/gst-plugins-base/.', '-IC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/.', '-IC:/gtk-build/build/x64/release/gst-plugins-base/gst-libs', '-IC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gst-libs', '--filelist=C:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gst-libs/gst/tag/gsttag-1.0-0.dll.p/GstTag_1.0_gir_filelist', '--include=Gst-1.0', '--include=GstBase-1.0', '--symbol-prefix=gst', '--identifier-prefix=Gst', '--pkg-export=gstreamer-tag-1.0', '--cflags-begin', '-DG_DISABLE_CAST_CHECKS', '-IC:/gtk-build/build/x64/release/gst-plugins-base/.', '-IC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/.', '-IC:/gtk-build/build/x64/release/gst-plugins-base/gst-libs', '-IC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gst-libs', '-IC:/gtk-build/gtk/x64/release/bin/../include', '-IC:/gtk-build/gtk/x64/release/bin/../include/gstreamer-1.0', '-IC:/gtk-build/gtk/x64/release/bin/../include/glib-2.0', '-IC:/gtk-build/gtk/x64/release/bin/../lib/glib-2.0/include', '-IC:/gtk-build/gtk/x64/release/bin/../include/gobject-introspection-1.0', '--cflags-end', '--add-include-path=C:/gtk-build/gtk/x64/release/bin/../share/gir-1.0', '-LC:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/gst-libs/gst/tag', '--library', 'gsttag-1.0', '-LC:/gtk-build/gtk/x64/release/bin/../lib', '-LC:/gtk-build/gtk/x64/release/bin/../lib', '--extra-library=gmodule-2.0', '--extra-library=gstbase-1.0', '--extra-library=gstreamer-1.0', '--extra-library=glib-2.0', '--extra-library=intl', '--extra-library=gobject-2.0', '--extra-library=zlib1', '--extra-library=girepository-1.0', '--sources-top-dirs', 'C:/gtk-build/build/x64/release/gst-plugins-base/', '--sources-top-dirs', 'C:/gtk-build/build/x64/release/gst-plugins-base/_gvsbuild-meson/']
--- stdout ---

--- stderr ---
Couldn't find include 'Gst-1.0.gir' (search path: '['C:/gtk-build/gtk/x64/release/bin/../share/gir-1.0', 'C:\\gtk-build\\gtk\\x64\\release\\share', 'gir-1.0', 'C:\\gtk-build\\gtk\\x64\\release\\share\\gir-1.0']')

[62/774] Compiling C object gst-libs/gst/video/gstvideo-1.0-0.dll.p/video-blend.c.obj
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\utils\builder.py", line 527, in build
    if self.__build_one(p):
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\utils\builder.py", line 663, in __build_one
    skip_deps = proj.build()
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\projects\gstreamer.py", line 108, in build
    Meson.build(self)
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\utils\base_builders.py", line 58, in build
    self.builder.exec_ninja(params="install", working_dir=ninja_build)
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\utils\builder.py", line 983, in exec_ninja
    self.__execute(
  File "d:\GitClones\gvsbuild\.venv\lib\site-packages\gvsbuild\utils\builder.py", line 1039, in __execute
    subprocess.check_call(args, cwd=working_dir, env=env, shell=True)
  File "C:\Users\Tim\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'ninja install' returned non-zero exit status 1.
Error: gst-plugins-base build failed
```
I'm going to and add `--clean` and see what happens.

--- a/gvsbuild/projects/gstreamer.py
+++ b/gvsbuild/projects/gstreamer.py
@@ -124,8 +124,6 @@ class GstPluginsGood(Tarball, Meson):
             dependencies=[
                 "meson",
                 "ninja",
-                "glib",
-                "gstreamer",
                 "gst-plugins-base",
                 "libvpx",
             ],
@@ -145,7 +143,7 @@ class GstPluginsBad(Tarball, Meson):
             version="1.20.5",
             archive_url="https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-{version}.tar.xz",
             hash="f431214b0754d7037adcde93c3195106196588973e5b32dcb24938805f866363",
-            dependencies=["meson", "ninja", "glib", "gstreamer", "gst-plugins-base"],
+            dependencies=["meson", "ninja", "gst-plugins-base"],
             patches=[
                 "wasapisink-reduce-buffer-latency.patch",
             ],
@@ -167,7 +165,7 @@ class GstPython(Tarball, Meson):
             version="1.20.5",
             archive_url="https://gstreamer.freedesktop.org/src/gst-python/gst-python-{version}.tar.xz",
             hash="27487652318659cfd7dc42784b713c78d29cc7a7df4fb397134c8c125f65e3b2",
-            dependencies=["meson", "ninja", "glib", "gstreamer", "pygobject"],
+            dependencies=["meson", "ninja", "pygobject", "gst-plugins-base"],
         )
 
     def build(self):

